mne/epochs.py

```diff
@@ -1389,6 +1402,42 @@ def save(self, fname):
         end_block(fid, FIFF.FIFFB_PROCESSED_DATA)
         end_block(fid, FIFF.FIFFB_MEAS)
         end_file(fid)
+        return use_fname
+
+    def save(self, fname, split_size='2GB'):
+        """Save epochs in a fif file
+
+        Parameters
+        ----------
+        fname : str
+            The name of the file, which should end with -epo.fif or
+            -epo.fif.gz.
+        split_size : string | int
+            Large raw files are automatically split into multiple pieces. This
+            parameter specifies the maximum size of each piece. If the
+            parameter is an integer, it specifies the size in Bytes. It is
+            also possible to pass a human-readable string, e.g., 100MB.
+            Note: Due to FIFF file limitations, the maximum split size is 2GB.
+        """
```

Comment:
We should add a `Notes` section that mentions that bad epochs will be dropped by this process because `get_data` is used